src/Microsoft.Data.Sqlite/SqliteConnectionStringBuilder.cs

```diff
@@ -15,11 +15,13 @@ public class SqliteConnectionStringBuilder : DbConnectionStringBuilder
     {
         private const string DataSourceKeyword = "Data Source";
         private const string CacheKeyword = "Cache";
+        private const string TimeoutKeyword = "Timeout";
```

Comment:
We shouldn't have a connection string setting for this. It changes over the lifetime of the connection, and there is already API for setting it: DbCommand.CommandTimeout.